_posts/loadbalancers/2024-02-26-introduction-into-loadbalancer.md

```diff
@@ -0,0 +1,124 @@
+
+### Session Persistence
+Session persistence is used to make sure that the traffic from the same client is always forwarded
+to the same backend server. This is used to make sure that the session data is always available on
+the same server. This is essential for applications that require session data to be available on the
+same server. OpenStack Octavia currently supports the following Session persistence methods:
+- **HTTP_COOKIE**: is used to make sure that the traffic from the same client is always forwarded to
+the same backend server based on the HTTP cookie.  
+- **APP_COOKIE**: is used to make sure that the traffic from the same client is always forwarded to
+the same backend server based on the application cookie.  
+- **SOURCE_IP**: is used to make sure that the traffic from the same client is always forwarded to
+the same backend server based on the source IP of the incoming traffic.  
+
+---
+
+## Monitors
+Monitors are used to define the health of the backend servers. The monitors are used to define the
+protocol, the interval, the timeout, the retries, and the status of the backend servers. Whenever a
+backend server or application is down, the monitor will mark the server as down and the traffic will
+not be forwarded to that server anymore. This way you can make sure that the traffic is only
+forwarded to the healthy servers to avoid downtime for the users.
+
+---
+
+## SSL Termination (HTTPS)
+SSL Termination is used to decrypt the incoming traffic and forward it to the backend servers. This
+is used to offload the SSL encryption from the backend servers. This way the backend servers can
+focus on the application and the loadbalancer can focus on the SSL encryption. The storage of the
+SSL certificates is be done with OpenStack Barbican (Secret Manager) and can be added to the
+loadbalancers on creation or at a later point. It is required to use the `TERMINATED_HTTPS` listener 
+protocol to enable SSL Termination at the Loadbalancer.
+
+---
+
+## Where to place your loadbalancer
+Chosing the best location for you loadbalancer can be an essential part of the setup. We recommend
+checking the geographical location of the backend servers to make sure the loadbalancer is placed in
+the desired region/availability zone (AZ).
+
+Loadbalancers can be used to forward external traffic to the backend servers but they may also be
+used to forward internal traffic within your private network. This is something to keep in mind when
+ desiging your cloud infrastructure.
+
+---
+
+## Recommendations
+When creating a loadbalancer setup, it is recommended to use the Active/Standby Loadbalancer setup.
+This is because of the reliability and availability of the setup. The Active/Standby Loadbalancer
+are setup to make sure the availability of the servers and applications can be assured whenever one
+of the loadbalancers fails.
+
+When you create a loadbalancer we recommend to use an Floating IP address for the loadbalancer.
```

Comment:
Should we refer to the article on how to create a load balancer here?

Reply:
We could, but I am not sure if we want to because we might want to add a more generic article on how to create one without the webservers etc.

Reply:
if so, we can always change the link again and the lb article currently doesn't have the creation of web servers in the same article, but also refers to another article

Reply:
Done